openpype/scripts/otio_burnin.py

```diff
@@ -573,8 +609,43 @@ def burnins_from_data(
             burnin.add_timecode(*args)
             continue
 
-        text = value.format(**data)
-        burnin.add_text(text, align, frame_start, frame_end)
+        cmd = ""
+        text = None
+        keys = [i[1] for i in Formatter().parse(value) if i[1] is not None]
+        list_to_convert = []
+
+        # Warn about nested dictionary support for lists. Ei. we dont support
+        # it.
+        if "[" in "".join(keys):
+            print(
+                "We dont support converting nested dictionaries to lists,"
+                " so skipping {}".format(value)
+            )
+        else:
+            for key in keys:
+                data_value = data[key]
+
+                # Multiple lists are not supported.
+                if isinstance(data_value, list) and list_to_convert:
+                    raise ValueError(
+                        "Found multiple lists to convert, which is not "
+                        "supported: {}".format(value)
+                    )
+
+                if isinstance(data_value, list):
+                    print("Found list to convert: {}".format(data_value))
+                    for v in data_value:
+                        data[key] = v
+                        list_to_convert.append(value.format(**data))
+
+        if list_to_convert:
+            text = list_to_convert[0]
+            cmd = convert_list_to_cmd(list_to_convert, 25.0, label=align)# need to fetch fps properly
```

Comment:
`fps` should not be hardcoded to 25.0

Reply:
I agree, hence the comment to fetch it properly :)

Reply:
Haha you working on draft PRs that I even end up reading too quickly.